components/package-template/src/etc/clp-config.yml

```
@@ -83,6 +83,9 @@
 #
 #  # How much data CLP should try to fit into each segment within an archive
 #  target_segment_size: 268435456  # 256 MB
+#  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)